### PR TITLE
LittlevGL --> LVGL

### PR DESCRIPTION
**Pull request by ropg**
New batch of LittlevGL --> LVGL changes. Note that in the LICENSE file I changed

> Copyright (c) 2019 Littlev Graphics Library

to

> Copyright (c) 2019-2021 LVGL

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [submodule "lvgl"]
 	path = lvgl
-	url = https://github.com/littlevgl/lvgl.git
+	url = https://github.com/lgl/lvgl.git
```
Comment (amirgon) [inline .gitmodules:3]: How did the test pass??
You have a typo: github.com/**lgl**/lvgl.git

Comment (embeddedt) [reply .gitmodules:3]: Submodules are updated from `lv_bindings` *before* the PR copy is checked out, so it uses whatever is on GitHub. That looks like an oversight in the CI.

**Comment by ropg** [comment]
WHOOPS...

Fixed...

**Comment by amirgon** [comment]
Merged.

Thanks!

**Pull request merged**